### ISSUE TITLE
spf13/vim-autoclose does not support ibus input method in gvim

### ISSUE THREAD

**Issue by sgon00** [spf13/spf13-vim#448]
spf13/vim-autoclose does not support ibus input method in gvim.
In terminal vim, everything works fine. 
In gvim, when I switch to ibus input method to input non-english words, the input method will quit after typing the first char. 
Currently, I unbundle this plugin.

**Comment by jannson** [comment]
Yes, I have the same problem. so I search from google and get here. thanks.

**Comment by jannson** [comment]
I use 
echo UnBundle \'AutoClose\' >> ~/.vimrc.bundles.local
:BundleClean!
but it not works.

**Comment by jannson** [comment]
finally I found  that it's because of auto-pairs!!!
